docs/source/model_design/general_reference_material/bliss/index.rst

```diff
@@ -604,27 +621,68 @@ guess of what specific sex- and gender-related factors could be associated with
 
   Reference 'measurement error' section again here.
 
-4.2 Further Results (Text, Figures, Tables)
--------------------------------------------
+4.4 Result indicator definition
+-------------------------------
 
-Of course, it is not possible to include a long explanation each and every time sex- or
-gender-related concepts are referenced.
-After an initial, comprehensive description, we have to switch to some kind of shorthand.
+In *each* stand-alone document (paper, table, figure, etc) where we present results
```

Comment:
does this concept of stand-alone document make sense to everyone?  I'm interpreting it to mean in the main paper, in each table, and in each figure, since any table or figure might be referred to in isolation by a busy scientist.

Reply:
I think this makes sense to me.

Reply:
Hm I might take a crack at spelling this out more explicitly regardless.